test/module/irohad/model/converters/pb_query_factory_test.cpp

```diff
@@ -38,13 +38,27 @@ void runQueryTest(std::shared_ptr<Query> query){
 }
 
 TEST(PbQueryFactoryTest, SerializeGetAccount){
+  auto createdTime = 111u;
```

Comment:
Pleas follow the [codestyle](http://google.github.io/styleguide/cppguide.html#Variable_Names), rename to `created_time`
Same for the rest consts